Packs/CortexXDR/Layouts/layoutscontainer-Cortex_XDR_Incident.json

```diff
@@ -842,6 +884,344 @@
+                                "fieldId": "country",
+                                "height": 22,
+                                "id": "4c8d5610-0f52-11ee-81b3-5b1a51073e91",
+                                "index": 3,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Attacker Extra Data",
+                        "static": false,
+                        "w": 1,
+                        "x": 1,
+                        "y": 0
+                    },
+                    {
+                        "h": 3,
+                        "i": "1vduzkpmlh-ohwq8lhsex-caseinfoid-944f47f0-3fce-11ed-81fb-f98f11f06b6f",
+                        "items": [],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Malicious or Suspicious Indicators",
+                        "query": "reputation:Benign OR reputation:Suspicious OR reputation:Malicious",
+                        "queryType": "input",
+                        "static": false,
+                        "type": "indicators",
+                        "w": 2,
+                        "x": 0,
+                        "y": 2
+                    },
+                    {
+                        "h": 2,
+                        "i": "1vduzkpmlh-ohwq8lhsex-caseinfoid-738a28d0-ffd3-11ed-94b9-ab17767bb4e7",
+                        "items": [],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Hunting Results",
+                        "query": {
+                            "categories": [
+                                "tags"
+                            ],
+                            "preDefinedFilters": true,
+                            "tags": [
+                                "PersistenceHunting"
+                            ]
+                        },
+                        "queryType": "warRoomFilter",
+                        "static": false,
+                        "type": "invTimeline",
+                        "w": 1,
+                        "x": 2,
+                        "y": 5
+                    },
+                    {
+                        "description": "The indicators extracted from the threat hunting results.",
+                        "h": 3,
+                        "i": "1vduzkpmlh-ohwq8lhsex-caseinfoid-494706a0-063c-11ee-b283-eb5063f641e1",
+                        "items": [],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Cloud Indicators",
+                        "query": "99900222-7570-4e56-8fa6-1206e76be060",
```

Comment:
we need to make sure that we dont need to change it to the script name (now its the ID)

Reply:
fixed